scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -307,9 +307,9 @@ void jobSubmitted(InternalJob job) {
         for (Map.Entry<JobId, JobData> entry : jobs.entrySet()) {
             JobData value = entry.getValue();
 
-            // If the scheduler is paused, schedule only running jobs
+            // If the scheduler is paused, schedule only running or stalled jobs
             if (isSchedulerPausedOrStopped &&
-                (value.job.getStatus() != JobStatus.RUNNING || value.job.getStatus() != JobStatus.STALLED)) {
```

Comment:
you can remove the parenthesis in the second statement.

Reply:
Good point.. yes sure, but I dont want since i think it helps to understand the condition.